diesel/src/expression/sql_literal.rs

```diff
@@ -94,5 +181,81 @@ impl<ST> NonAggregate for SqlLiteral<ST> {}
 /// # }
 /// ```
 pub fn sql<ST>(sql: &str) -> SqlLiteral<ST> {
-    SqlLiteral::new(sql.into())
+    SqlLiteral::new(sql.into(), ())
+}
+
+#[derive(Debug, Clone, Copy)]
+#[must_use = "Queries are only executed when calling `load`, `get_result`, or similar."]
+pub struct UncheckedBind<Query, Value> {
+    query: Query,
+    value: Value,
+}
+
+impl<Query, Value> UncheckedBind<Query, Value>
+where
+    Query: Expression,
+{
+    pub fn new(query: Query, value: Value) -> Self {
+        UncheckedBind { query, value }
+    }
+
+    pub fn sql(self, sql: &str) -> SqlLiteral<Query::SqlType, Self> {
```

Comment:
Needs docs.

Reply:
@sgrif Do we need this method? If we don't plan on supporting`.filter(sql("id > 1").sql(" AND name <> 'Ryan'"))...` I think I can omit this.